tests/test_spstats.py

```diff
@@ -66,6 +67,30 @@ def test_empirical_fit_variogram_running(self):
         fun2, _ = xdem.spstats.fit_model_sum_vgm(['Sph','Sph','Sph'],emp_vgm_df=df_sig)
         xdem.spstats.plot_vgm(df_sig,fit_fun=fun2)
 
+    def test_multirange_fit_performance(self):
+
+        # first, generate a true sum of variograms with some added noise
+        r1, ps1, r2, ps2, r3, ps3 = (100,0.7,1000,0.2,10000,0.1)
+
+        x = np.linspace(10,20000,500)
+        y = models.spherical(x,r=r1,c0=ps1) + models.spherical(x,r=r2,c0=ps2) \
+            + models.spherical(x,r=r3,c0=ps3)
+
+        sig = 0.025
+        y_noise = np.random.normal(0,sig,size=len(x))
+
+        y_simu = y + y_noise
+        sigma = np.ones(len(x))*sig
+
+        df = pd.DataFrame()
```

Comment:
Will instantiating DataFrames without a dtype not trigger a warning since a specific Pandas version? Or is that maybe just for Series maybe...

Reply:
No warnings ;)

Reply:
I just tried and it's fine in my version.